lib/vscode/src/vs/base/common/product.ts

```diff
@@ -26,6 +26,9 @@ export type ExtensionUntrustedWorkspaceSupport = {
 };
 
 export interface IProductConfiguration {
+	// NOTE@coder: add codeServerVersion
+	readonly codeServerVersion?: string;
```

Comment:
Does this mean we won't show the code-server version in the `Help > About`?

Reply:
Showing the code-server version still works - it was just moved between two files (see the commit that includes this change), so I moved it here.